solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -15,8 +15,12 @@
 
 # Titles to refer to counts of preprocessing results
 VALIDATION_RESULT_TOTAL_STRING = "TOTAL FLAGGED VALUES DISCARDED"
+FILL_RESULT_TOTAL_STRING = "Total {0}Forecast Values {1}"
```

Comment:
```suggestion
FILL_RESULT_TOTAL_STRING = "Total {0} Forecast Values {1}"
```

Reply:
This produces an extra space because I want the output to be:
- `Total Forecast Values ...` for the original forecast and
- `Total Reference Forecast Values ...` for a reference forecast

Of course multiple ways to address this `Total {0} Values {1}` for instance. Whatever your preference is fine with me.